src/core/src/op/grid_sample.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/grid_sample.hpp"
```

Comment:
I see your commit https://github.com/openvinotoolkit/openvino/pull/11770/commits/e9d130d229bc1841058a1d8a6327eb6a7aae7820, but including openvino shouldn't be an issue here.
Reference: https://github.com/openvinotoolkit/openvino/pull/11285#discussion_r838159742
```suggestion
#include "openvino/op/grid_sample.hpp"
```